### ISSUE TITLE
FR: add discount

### ISSUE THREAD

**Issue by bartrylant** [KuriousAgency/commerce-admin-orders#28]
I've asked this in another issue already, but I'd like to bring it up again. It would be nice if there would be an extra line just to give a variable discount. I know you can edit it afterwards, but it's a strange flow. My client is using this plugin for sales in the real shop. She now has to make a new order, add items, continue, continue, continue and add a payment. Only after that step, she can edit the line items to add a discount, recalculate the complete order, refund the old order and pay again. It would make more sense if you can just add a discount in the beginning, just like adding an item to the cart or a coupon code.
